cfpb/consumerfinance.gov
# Archiving: add archive pill to filterable list view

Adds an archive pill to post previews in a filterable list view.

---

## Additions

- Archived pill in post preview template


## Changes

- Update archived pill template to accept a page object


## How to test this PR

1. Check out branch and run gulp styles
2. In Wagtail admin, mark a page under Research & Reports and one under Blog as archived
3. Navigate to the [blog](http://localhost:8000/about-us/blog/?title=&from_date=&to_date=&archived=include) and [Research & Reports](http://localhost:8000/data-research/research-reports/?title=&from_date=&to_date=&archived=include) pages with the "show all items" archived filter option checked
4. Verify that archived pill shows in post preview for archived pages
5. Verify that archived pill shows correctly in the archived page's header


## Screenshots
### Filterable list view
<img width="648" alt="Screen Shot 2020-11-04 at 9 46 41 AM" src="https://user-images.githubusercontent.com/778171/98149936-b9664080-1e82-11eb-97e8-848e28d4d3b3.png">

### Archived page view

<img width="838" alt="Screen Shot 2020-10-26 at 12 08 40 PM" src="https://user-images.githubusercontent.com/778171/97217920-2d1e9400-1785-11eb-90b3-8c6f4de4c7e0.png">
